app/views/projects/_project_assignment_form.html.erb

```diff
@@ -1,8 +1,7 @@
 <%= form_with model: project, url: assign_project_path(project), method: :patch, class: 'form-inline' do |form| %>
   <div class="form-group">
-    <%= form.label :assigned_user_id, 'Application Manager: ' do %>
-      <%= form.collection_select :assigned_user_id, User.all_application_managers.in_use, :id, :full_name, { include_blank: true }, class: 'form-control' %>
-    <% end %>
+    <%= form.label :assigned_user_id, 'Application Manager' %>
+    <%= form.collection_select :assigned_user_id, User.all_application_managers.in_use, :id, :full_name, { include_blank: true, disabled: project.assigned_user_id }, class: 'form-control' %>
```

Comment:
currently you can still hit the apply button when the disabled option is preselected when loading the page, so we get an missing params value error